rule-engine/rule-engine-components/src/main/java/org/thingsboard/rule/engine/transform/TbChangeOriginatorNode.java

```diff
@@ -112,27 +103,19 @@ private ListenableFuture<? extends EntityId> getNewOriginator(TbContext ctx, TbM
     }
 
     private void validateConfig(TbChangeOriginatorNodeConfiguration conf) {
-        HashSet<String> knownSources = Sets.newHashSet(CUSTOMER_SOURCE, TENANT_SOURCE, RELATED_SOURCE, ALARM_ORIGINATOR_SOURCE, ENTITY_SOURCE);
-        if (!knownSources.contains(conf.getOriginatorSource())) {
-            log.error("Unsupported source type '{}'! Only {} types are allowed.", conf.getOriginatorSource(), supportedOriginatorSourcesStr);
-            throw new IllegalArgumentException("Unsupported source type '" + conf.getOriginatorSource() +
-                    "'! Only " + supportedOriginatorSourcesStr + " types are allowed.");
-        }
-
-        if (conf.getOriginatorSource().equals(RELATED_SOURCE)) {
+        if (conf.getOriginatorSource().equals(RELATED)) {
             if (conf.getRelationsQuery() == null) {
-                log.error("Relations query should be specified if 'Related entity' source is selected.");
+                log.debug("Relations query should be specified if 'Related entity' source is selected.");
```

Comment:
merge if statements. In addition, please add the check for null for: conf.getOriginatorSource()